website/content/docs/contributor-guide.md

```diff
@@ -1,11 +1,37 @@
 ---
 title: Contributor Guide
-weight: 100
+weight: 20
 group: contributing
 ---
 
 We're hoping that Netlify CMS will do for the [JAMstack](https://www.jamstack.org) what WordPress did for dynamic sites back in the day. We know we can't do that without building a thriving community of contributors and users, and we'd love to have you join us.
 
+## Getting started with contributing
+Being a developer is not a requirement for contributing to Netlify CMS, you only need the desire, a web browser, and a [GitHub account](https://github.com/join). The GitHub repo has a step-by-step [guide](https://github.com/netlify/netlify-cms/blob/master/CONTRIBUTING.md) to get started with the code.
+
+## The basics of the Netlify CMS docs
+The documentation for Netlify CMS is written in [Markdown](http://daringfireball.net/projects/markdown/) (a good cheatsheet on Markdown is [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)), with the source residing on [GitHub](https://github.com/netlify-cms) in the `/website/content/docs` folder.
+
+The GitHub website allows you to submit issues, work with files, search for content, and browse changes that have been submitted in the past and those that are being submitted now (aka Pull Requests). 
+
+## Style guidelines
+A [style guide](/website/content/docs/style-guide.md) is available to help provide context around grammar, code styling, syntax, etc. 
+
+## Filing issues
+If you have a GitHub account, you can file an [issue](https://github.com/netlify/netlify-cms/issues) (aka bug report) against the Netlify CMS docs. Even if you're not able to, or don't know how to, fix the issue (see [Improve existing content](#improve-existing-content)), it helps to start the conversation. When you file an issue, it's important to include the following info:
```

Comment:
We can remove the final sentence of this paragraph, the list of items to include, and the TODO note, as users opening a new issue will be given a choice between bug and feature issue templates, both of which provide detailed information about what to include.